arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -163,19 +165,41 @@ public List<ReplayCompareResult> doContentCompare(ReplayActionCaseItem caseItem,
       List<CategoryComparisonHolder> waitCompareMap) {
     ComparisonInterfaceConfig operationConfig = compareConfigService.loadInterfaceConfig(
         caseItem.getParent());
-    List<String> ignoreCategoryList = operationConfig.getIgnoreCategoryTypes();
+    List<CategoryDetail> ignoreCategoryList = operationConfig.getIgnoreCategoryTypes();
 
     List<ReplayCompareResult> replayCompareResults = new ArrayList<>();
     for (CategoryComparisonHolder bindHolder : waitCompareMap) {
       if (operationConfig.checkIgnoreMockMessageType(bindHolder.getCategoryName(),
-          ignoreCategoryList)) {
+          caseItem.getParent().getOperationName(), ignoreCategoryList)) {
         continue;
       }
+      filterIgnoredCompareItem(bindHolder.getReplayResult(), operationConfig);
```

Comment:
suggestion: put the filtering of record and replay data in one method

Reply:
accepted